src/referencePackages/src/microsoft.win32.primitives/4.3.0/Microsoft.Win32.Primitives.csproj

```diff
@@ -15,7 +15,6 @@
     <Compile Include="**/ref/$(TargetFramework)/*.cs" />
     <Compile Include="**/lib/$(TargetFramework)/*.cs" />
   </ItemGroup>
-
     <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
```

Comment:
Stray whitespace change? This blank section separator is one that I actually expect to be there.

(On the other hand, it would be nice to do some cleanup of the mixed 2-space vs. 4-space tabs and stuff like that, but without tooling being tied to the repo version (so we can fix the source of the formatting issues as well) it's probably not the right time to try that.)

Reply:
I updated the generator to generate the projects and corrected the 2-space vs. 4-space issue as well as spacing between sections.  Now all generated projects will have 2-space tabs with one space between sections.  This will only be for projects that are new or are re-generated because they are dependencies.